### PR TITLE
feat(BreadCrumb): add BreadCrumb component

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

File: app/src/components/index.ts [hunk 1]
```diff
@@ -13,3 +13,4 @@ export * from "./Input";
 export * from "./TagsInput";
 export * from "./FileUploader";
 export * from "./Pagination";
+import * from "./BreadCrumb";
```
Comment (HQFOX) [inline app/src/components/index.ts:16]: this should be an export

File: packages/core/src/components/BreadCrumb/Page/pageClasses.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { getClasses } from "utils";
+
+export type HvPageClasses = {
```
Comment (MEsteves22) [inline packages/core/src/components/BreadCrumb/Page/pageClasses.ts:3]: It should be possible for the classes to be undefined as we discussed

File: packages/core/src/components/BreadCrumb/PathElement/pathElementClasses.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { getClasses } from "utils";
+
+export type HvPathElementClasses = {
+  centerContainer: string;
```
Comment (MEsteves22) [inline packages/core/src/components/BreadCrumb/PathElement/pathElementClasses.ts:4]: It should be possible for the classes to be undefined as we discussed

File: packages/core/src/components/BreadCrumb/breadCrumbClasses.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { getClasses } from "utils";
+
+export type HvBreadCrumbClasses = {
+  root: string;
```
Comment (MEsteves22) [inline packages/core/src/components/BreadCrumb/breadCrumbClasses.ts:4]: It should be possible for the classes to be undefined as we discussed

File: packages/core/src/components/BreadCrumb/BreadCrumb.tsx [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+      : listPath;
+
+  return (
+    <StyledRoot
+      id={id}
+      className={clsx(breadCrumbClasses.root, classes?.root, className)}
+      {...others}
+    >
+      <StyledOrderedList className={classes?.orderedList}>
+        {listPath.map((elem, index) => {
```
Comment (MEsteves22) [inline packages/core/src/components/BreadCrumb/BreadCrumb.tsx:94]: Missing class from `breadCrumbClasses`.

File: packages/core/src/components/BreadCrumb/BreadCrumb.tsx [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+          const key = `key_${index}`;
+          const isLast = index === breadcrumbPath.length - 1;
+
+          {
+            return (
+              <HvPathElement
+                classes={{
+                  centerContainer: classes?.centerContainer || "",
+                  separatorContainer: classes?.separatorContainer || "",
```
Comment (MEsteves22) [inline packages/core/src/components/BreadCrumb/BreadCrumb.tsx:103]: Missing class from `breadCrumbClasses`.

File: packages/core/src/components/BreadCrumb/BreadCrumb.tsx [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+                }}
+                key={key}
+                last={isLast}
+              >
+                {(isValidElement(elem) && elem) ||
+                  (isLast && (
+                    <HvTypography
+                      className={classes?.currentPage}
```
Comment (MEsteves22) [inline packages/core/src/components/BreadCrumb/BreadCrumb.tsx:111]: Missing class from `breadCrumbClasses`.

File: packages/core/src/components/BreadCrumb/Page/Page.tsx [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+      as={Component}
+      onClick={onClick}
+      data={elem}
+      classes={{ a: clsx(pageClasses.a, classes?.a) }}
+    >
+      <StyledTypography
+        // noWrap
+        as="div"
+        variant="label"
+        className={clsx(pageClasses.link, classes?.link)}
```
Comment (MEsteves22) [inline packages/core/src/components/BreadCrumb/Page/Page.tsx:36]: The [label styling](https://github.com/lumada-design/hv-uikit-react/blob/master/packages/core/src/BreadCrumb/styles.js#L40) is missing.

**Review by HQFOX — APPROVED** [review]

**Review by MEsteves22 — APPROVED** [review]

**Pull request merged**